k8s/aat/common-overlay/sscs/kustomization.yaml

```diff
@@ -4,6 +4,9 @@ namespace: sscs
 bases:
 - ../../../namespaces/sscs
 - ../../../namespaces/all-namespaces/nonprod-role.yaml
+- ../../../namespaces/sscs/sscs-hearings-api/sscs-hearings-api.yaml
```

Comment:
I am assuming once all apps are kustomized - you will be moving this into namespaces/sscs - rather thn a specific per each app